content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -182,44 +213,93 @@ Run the following commands on the virtual machine you want to add to the Istio m
     $ sudo apt -y upgrade
     {{< /text >}}
 
+1. Install the root cert to the `/var/run/secrets/istio`
+    
+    {{< text bash >}}
+        $ sudo mkdir -p /var/run/secrets/istio
+        $ sudo cp ~/root-cert.pem /var/run/secrets/istio/root-cert.pem
+    {{< /text >}}
+    
+1. Install the token to the `/var/run/secrets/tokens`
```

Comment:
don't use `~' - many people people don't know what it means. please use `"${HOME}".